### PR TITLE
Hello rails back end

### PR DESCRIPTION
**Pull request by binyamolango**
# Hi Reviewer 👋
Below is a list of the accomplished tasks while working on this project:

- Created a feature branch.
- Added a workflow to the project.
- Implemented Rubocop linters.
- Generated the Greeting model.
- Ran migration file.
- Generated the API controller.
- Defined index action for API controller.
- Set up root route.
- Implemented the rack-cors dependencies.
- Fixed linters' errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,121 @@
...
-* ...
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+To use this project, Clone this repository to your desired folder by pasting this command in your command line interface:
+
+  - `git clone https://github.com/binyamolango/hello-rails-back-end.git`
+  - `git clone https://github.com/binyamolango/hello-react-front-end.git`
+
```
Comment (mckent05) [inline README.md]: - You have done well so far, though I can see the link to your frontend here, but according to the project requirements you should have a link that leads directly to the frontend repo, please fix this 👍 👍

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
+  resources :greetings
```
Comment (mckent05) [inline config/routes.rb]: - You have done well so far, but please note that implementing your routes this way exposes all endpoints, leaving your app prone to attacks. I suggest exposing only the index action that is in use 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @binyamolango, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job implementing your rails API 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @binyamolango, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Fixed the link issue in the README file ✅ 
- Update routes by adding `index` only attribute ✅ 

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**